lib/success.js

```diff
@@ -48,12 +48,14 @@ module.exports = async (pluginConfig, context) => {
       async q => (await github.search.issuesAndPullRequests({q})).data.items
     );
 
-    const prs = await pFilter(
-      uniqBy(flatten(await Promise.all(searchQueries)), 'number'),
-      async ({number}) =>
-        (await github.pullRequests.listCommits({owner, repo, number})).data.find(({sha}) => shas.includes(sha)) ||
-        shas.includes((await github.pullRequests.get({owner, repo, number})).data.merge_commit_sha)
-    );
+    const prs = await pFilter(uniqBy(flatten(await Promise.all(searchQueries)), 'number'), async ({number}) => {
+      const options = {owner, repo, pull_number: number}; // eslint-disable-line camelcase
+
+      return (
+        (await github.pullRequests.listCommits(options)).data.find(({sha}) => shas.includes(sha)) ||
+        shas.includes((await github.pullRequests.get(options)).data.merge_commit_sha)
+      );
+    });
```

Comment:
changed the style a bit to avoid having two `// eslint-disable-line camelcase` comments